example.csharp/addons/tiltfive/scenes/T5ControllerCS.cs

```diff
@@ -0,0 +1,25 @@
+using Godot;
+using System;
+
+public partial class T5ControllerCS : Node3D
```

Comment:
Would it be possible to declare C# friendly signals here. The _ready can do the ugly Connect by strings and fire the C# delegates.

Reply:
Yeah, I thought about that but I wasn’t sure I wanted the second level of indirection but it’s probably not a big deal.